packages/babel-preset-env/data/shipped-proposals.js

```diff
@@ -3,12 +3,15 @@
 
 const proposalPlugins = {};
 
+// Please keep these in alphabetical order!
```

Comment:
If you want to enforce these tuples being in alphabetical order, you could use `/* eslint sort-keys: "error" */` like you do in `available-plugins.js`, and create an object representing these plugins where the order will be enforced. 

Then you could transform it for the `new Map()`:
```
const pluginSyntaxMapObject = {
  "proposal-async-generator-functions": "syntax-async-generators",
  "proposal-json-strings": "syntax-json-strings",
  "proposal-nullish-coalescing-operator": "syntax-nullish-coalescing-operator",
  "proposal-object-rest-spread": "syntax-object-rest-spread",
  "proposal-optional-catch-binding": "syntax-optional-catch-binding",
  "proposal-optional-chaining": "syntax-optional-chaining",
  "proposal-unicode-property-regex": null,
}

const pluginSyntaxMap = new Map(Object.entries(pluginSyntaxMap))
```

Reply:
Good call - updated. I also resolved new conflicts with master.